src/Model/ChatMessage.php

```diff
@@ -248,15 +248,15 @@ public function setEtag($val)
     * Gets the from
     * Read only. Details of the sender of the chat message.
     *
-    * @return IdentitySet|null The from
+    * @return ChatMessageFromIdentitySet|null The from
     */
     public function getFrom()
```

Comment:
I think we have a breaking change here and in other `ChatMessage`-related models further on...

Scenario: a developer is currently passing an `IdentitySet` to `setFrom()`. If the call `getFrom()` they get back the `IdentitySet` object as expected.

With this change, they would get back a `ChatMessageFromIdentitySet` object that has been initialised with an `IdentitySet` object as opposed to a dictionary, meaning the getters on the returned object will return `null`.

Should we add this check for backward compatibility?
```php
if (is_a($this->_propDict['from"], "\Microsoft\Graph\Model\IdentitySet")) {
    $this->_propDict["from"] = new ChatMessageFromIdentitySet($this->_propDict["from"]->getProperties());
    return $this->_propDict["from"];
}
```

Reply:
Yes, this probably a breaking change.

Reply:
We shouldn't take this as there is a breaking change.

IdentitySet --> ChatMessageFromIdentitySet

We are waiting for word on more breaking change.

Reply:
Ok, closing this